src/main/resources/application.properties

```diff
@@ -33,6 +33,21 @@ orchestration.username=
 orchestration.password=
 orchestration.name=int
 
+# AWS Cloud Credentials
+# Instance Profile uses the credentials of the AWS instance
+cloud.aws.credentials.instanceProfile=true
+
+# Don't try to discover AWS stack information
+cloud.aws.stack.auto=false
+cloud.aws.region.static=us-east-1
+
+# Storage configuration for UI state information
+ui-state.storage.readonly=true
```

Comment:
I think this can default to false to that it works out of the box.

Reply:
Fixed